### PR TITLE
tools: support scan-status command and imporve status command

### PR DESCRIPTION
**Pull request by Wine93**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Wine93** [comment]
recheck

**Comment by Wine93** [comment]
recheck

**Comment by Wine93** [comment]
recheck

File: src/mds/topology/topology_service_manager.cpp [hunk 1]
```diff
@@ -1458,6 +1488,15 @@ void TopologyServiceManager::ListUnAvailCopySets(
     response->set_statuscode(kTopoErrCodeSuccess);
 }
 
+void TopologyServiceManager::ConvertCopyset(const CopySetInfo& in,
```
Comment (SeanHai) [inline src/mds/topology/topology_service_manager.cpp:1491]: Can use protobuf's 'CopyFrom'.

File: src/tools/copyset_check_core.cpp [hunk 2]
```diff
@@ -476,9 +477,36 @@ int CopysetCheckCore::CheckCopysetsWithMds() {
         std::cout << std::endl;
         ret = -1;
     }
+
+    // Check scan status for inconsistent copyset
+    auto nInconsistent = CheckScanStatus(copysetsInMds);
+    if (nInconsistent > 0) {
```
Comment (wu-hanqing) [inline src/tools/copyset_check_core.cpp:483]: Can you print detailed info about which copyset/chunk and which part of data is inconsistent?

File: src/mds/topology/topology_service_manager.cpp (hunk 1 above)
Comment (Wine93) [reply src/mds/topology/topology_service_manager.cpp:1491]: > Can use protobuf's 'CopyFrom'.

But the type of param `in` isn't a proto type

**Comment by Wine93** [comment]
recheck

File: src/tools/copyset_check_core.cpp (hunk 2 above)
Comment (Wine93) [reply src/tools/copyset_check_core.cpp:483]: > Can you print detailed info about which copyset/chunk and which part of data is inconsistent?

Actually, the detail will been print in [CopysetCheck::PrintDetail](https://github.com/opencurve/curve/blob/5ae2618538285503c04aaf287f6a2a0480acd68f/src/tools/copyset_check.cpp#L258) if you add the detail in `copysets_` map (I has added the detail into it)

File: proto/BUILD [hunk 3]
```diff
@@ -97,11 +97,10 @@ proto_library(
...
+    deps = [":scan_proto"],
 )
 
 proto_library(
     name = "scan_proto",
-    srcs = ["scan.proto"],        
-)
-
+    srcs = ["scan.proto"],
+)
```
[the remark this answers is not in the document]
Comment (Wine93) [reply proto/BUILD:106]: > A black line is needed at end of file.

done

File: proto/schedule.proto [hunk 4]
```diff
@@ -46,4 +46,4 @@ message QueryChunkServerRecoverStatusResponse {
 service ScheduleService {
     rpc RapidLeaderSchedule(RapidLeaderScheduleRequst) returns (RapidLeaderScheduleResponse);
     rpc QueryChunkServerRecoverStatus(QueryChunkServerRecoverStatusRequest) returns (QueryChunkServerRecoverStatusResponse);
-}
+}
```
[the remark this answers is not in the document]
Comment (Wine93) [reply proto/schedule.proto:49]: > Same

done

**Review by wu-hanqing — APPROVED** [review]

**Review by SeanHai — APPROVED** [review]

**Comment by Wine93** [comment]
recheck

**Comment by Wine93** [comment]
recheck

**Comment by Wine93** [comment]
recheck

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**